<launch>
    <node pkg="laser_filters" type="laserscanmultifilter_node" output="screen" name="laserscanmultifilter_node">
        <rosparam command="load" file="$(find laser_filters)/config/profiles_multifilter_test.yaml" />
        <param name="laserscan_topics" value="/scan1 /scan2 "/>  <!-- list of laser scan topics to filter and merge -->
        
        <param name="merged_destination_topic" value="/scan_merged"/>

        <param name="parent_frame" value="base_link"/>  
        <param name="destination_frame" value="laser_merged"/>  
        <rosparam param="transform"> [0, 0, 0, 0, 0, 0] </rosparam>

        <param name="angle_min" value="-3.14159"/>
        <param name="angle_max" value="3.14159"/>
        <param name="angle_increment" value="0.0058"/>
        <param name="scan_time" value="0.0333333"/>
        <param name="range_min" value="0.25"/>
        <param name="range_max" value="25.0"/>
    </node>

</launch>